clients/src/main/java/org/apache/kafka/common/network/Selector.java

```diff
@@ -858,46 +858,56 @@ public SelectorMetrics(Metrics metrics, String metricGrpPrefix, Map<String, Stri
             }
 
             this.connectionClosed = sensor("connections-closed:" + tagsSuffix.toString());
-            MetricName metricName = metrics.metricName("connection-close-rate", metricGrpName, "Connections closed per second in the window.", metricTags);
-            this.connectionClosed.add(metricName, new Rate());
+            MetricName rateMetricName = metrics.metricName("connection-close-rate", metricGrpName, "Connections closed per second in the window.", metricTags);
+            MetricName totalMetricName = metrics.metricName("connection-close-total", metricGrpName, "Total count of connections closed.", metricTags);
+            this.connectionClosed.add(new Meter(rateMetricName, totalMetricName));
```

Comment:
Can we not use the same approach we used in the consumer code?

Reply:
I was a bit concerned about this because the descriptions were not very consistent across metrics and I didn't want to lose anything from the description. Have updated.